lib/rack/urlmap.rb

```diff
@@ -75,7 +80,6 @@ def call(env)
       env[SCRIPT_NAME] = script_name
     end
 
-    private
```

Comment:
Any reason why `casecmp?` was made public? It appears to be used only in this class.

Reply:
Ah, typo. 

Originally moved the contents of `@mapping.each` out to its own private method, but this would have added some unnecessary complexity.